packages/preview/mahjong-tiles/0.1.0/typst.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "mahjong-tiles"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["hongjr03 <https://github.com/hongjr03>"]
+description = "Riichi-hand-rs but in Typst."
```

Comment:
In my opinion, this description could be improved. I personally don't know what riichi-hand-rs is, and I believe a lot of people don't either. Something along the lines of "Render Mahjong tiles" would be better.